src/main/scripts/main.py

```diff
@@ -16,7 +16,8 @@ def startAnalysis(analysisType, df, tweetsList, regressionMethod='1'):
     elif analysisType == "3":
         return analyzer.getEmbeddedAnalysis(regressionMethod=regressionMethod)
     elif analysisType == "4":
-        return analyzer.getCombinationAnalysis(tweetsList, regressionMethod=regressionMethod)
+        # return analyzer.getCombinationAnalysis(tweetsList, regressionMethod=regressionMethod)
```

Comment:
You can remove this comment too. It's useless now.
Also, I can't comment on the lines, but on lines 27 and 28, go ahead and remove that code. That was only used for the azure, but is now longer needed. 
```
print("Just a few more things to collect...")
allTweetsTextDocumentInputsList = nba_commisioner.getAllTweetsAllPlayersAsTextDocumentInputList()
```

Reply:
done